src/AutoRest.CSharp.V3/Generation/Types/TypeFactory.cs

```diff
@@ -99,6 +99,7 @@ private static bool IsList(CSharpType type)
             AllSchemaTypes.Duration => typeof(TimeSpan),
             AllSchemaTypes.OdataQuery => typeof(string),
             AllSchemaTypes.String => typeof(string),
+            AllSchemaTypes.Time => typeof(TimeSpan),
```

Comment:
This is absolute time, and should be a `DateTimeOffset`. See more thorough explanation below.

Reply:
It's not absolute, it's relative to the beginning of some day.